apps/full-site-editing/full-site-editing-plugin/full-site-editing/blocks/post-content/index.js

```diff
@@ -39,8 +39,10 @@ const addContentSlotClassname = createHigherOrderComponent( BlockListBlock => {
 	};
 }, 'addContentSlotClassname' );
 
+// Must be 9 or this breaks on Simple Sites
```

Comment:
Can't argue with that. 😄

![magic](https://user-images.githubusercontent.com/1182160/63047963-c1307280-bed5-11e9-8ac0-846259602cb0.gif)